app/src/main/java/com/infomaniak/swisstransfer/ui/MainActivity.kt

```diff
@@ -48,6 +49,7 @@ class MainActivity : ComponentActivity() {
                 MainScreen()
             }
         }
+        trackScreen()
```

Comment:
Shouldn't this be added in `NewTransferActivity` as well ?

Reply:
It was just for a test but ultimately, yes.